### PR TITLE
Python 3.8 support

### PR DESCRIPTION
**Pull request by guedou**
This PR _adds_ support for Python3.8.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,6 +40,12 @@ matrix:
           env:
             - TOXENV=py37-linux_non_root,codecov
 
+        - os: linux
+          dist: xenial
+          python: 3.8
```
Comment (gpotter2) [inline .travis.yml]:
```suggestion
          python: 3.7
```

File: .travis.yml [hunk 2]
```diff
@@ -94,6 +100,12 @@ matrix:
           env:
             - TOXENV=py37-linux_root,codecov
 
+        - language: python
+          dist: xenial
+          python: 3.8
```
Comment (gpotter2) [inline .travis.yml]:
```suggestion
          python: 3.7
```

**Comment by guedou** [comment]
I don't get your comment. I must be too tired =/

**Comment by gpotter2** [comment]
My bad I mixed things up.

**Comment by p-l-** [comment]
Looks like Travis was not entirely ready... Now they are (I just had to restart the two Python 3.8 tests).

**Pull request merged**